tests/test_uploads.py

```diff
@@ -80,6 +80,7 @@ def testGet(self):
         self.assertEqual(rv.status_code, 404)
 
     def testRegularUploads(self):
+        '''Test that regular file should not be possible to upload.'''
```

Comment:
This isn't quite accurate, if you look at what the code is doing.

What it does (in a loop), is try to upload some *bad* files  and some *good* files, and then asserts that it got the expected HTTP response code.

Unfortunately, there is a comment 2 lines down that says the wrong thing, so this is my fault.